src/back/space/src/api/create.cpp

```diff
@@ -0,0 +1,91 @@
+#include "create.hpp"
+#include "../service/service.hpp"
+#include "../../../shared/headers.hpp"
+#include <regex>
+
+namespace svetit::space::handlers {
+
+Create::Create(
+	const components::ComponentConfig& conf,
+	const components::ComponentContext& ctx)
+	: server::handlers::HttpHandlerJsonBase{conf, ctx}
+	, _s{ctx.FindComponent<Service>()}
+{}
+
+formats::json::Value Create::HandleRequestJsonThrow(
+	const server::http::HttpRequest& req,
+	const formats::json::Value& body,
+	server::request::RequestContext&) const
+{
+	formats::json::ValueBuilder res;
+
+	// todo - need to validate that this is valid uuid? need to check that user exists?
+	const auto& userId = req.GetHeader(headers::kUserId);
+	if (userId.empty()) {
+		res["err"] = "Empty userId header";
```

Comment:
Желательно чтобы текст который светим во внешний мир, содержал только общую информацию без подробностей реализации.
```suggestion
		res["err"] = "Access denied";
```

Reply:
Готово (поправил везде).